src/Node.h

```diff
@@ -7,10 +7,17 @@
 
 class Node
 {
-public:
+  public:
+    Node();
+
     Node(std::string startLocation);
 
+    bool operator==(Node zone);
+
+    void setNeighbor(Node* new_node, Edge* new_edge);
```

Comment:
are these the functions nisha and I wrote? if so, we seriously need documentation